pkgs/development/python-modules/python-djvulibre/default.nix

```
@@ -0,0 +1,53 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, cython
+, djvulibre
+, ghostscript
+, packaging
+, pkg-config
+, unittestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "python-djvulibre";
+  version = "0.9.0";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```
Could you please `pyproject = true` instead?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters

Then `setuptools` and `wheel` should be added to `nativeBuildInputs`.
https://github.com/FriedrichFroebel/python-djvulibre/blob/0609966fad04b7661488d59638f85862ef8a2708/pyproject.toml#L3